drivers/gles3/rasterizer_storage_gles3.h

```diff
@@ -1186,71 +1183,49 @@ class RasterizerStorageGLES3 : public RendererStorage {
 			};
 
 			Vector<Size> sizes;
-			GLuint color;
-			int levels;
+			GLuint color = 0;
+			int levels = 0;
 
-			MipMaps() :
-					color(0),
-					levels(0) {
+			MipMaps() {
 			}
 		};
 
 		MipMaps mip_maps[2];
 
 		struct External {
-			GLuint fbo;
-			GLuint color;
-			GLuint depth;
+			GLuint fbo = 0;
+			GLuint color = 0;
+			GLuint depth = 0;
 			RID texture;
 
-			External() :
-					fbo(0),
-					color(0),
-					depth(0) {
+			External() {
 			}
 		} external;
 
-		int x, y, width, height;
+		int x = 0, y = 0, width = 0, height = 0;
 
 		bool flags[RENDER_TARGET_FLAG_MAX];
 
 		// instead of allocating sized render targets immediately,
 		// defer this for faster startup
 		bool allocate_is_dirty = false;
-		bool used_in_frame;
-		RS::ViewportMSAA msaa;
+		bool used_in_frame = false;
+		RS::ViewportMSAA msaa = RS::VIEWPORT_MSAA_DISABLED;
 
-		bool use_fxaa;
-		bool use_debanding;
+		bool use_fxaa = false;
+		bool use_debanding = false;
 
 		RID texture;
 
-		bool used_dof_blur_near;
-		bool mip_maps_allocated;
+		bool used_dof_blur_near = false;
+		bool mip_maps_allocated = false;
 
 		Color clear_color;
-		bool clear_requested;
+		bool clear_requested = false;
 
 		RenderTarget() :
-				fbo(0),
-				color(0),
-				depth(0),
-				multisample_fbo(0),
-				multisample_color(0),
-				multisample_depth(0),
-				multisample_active(false),
-				x(0),
-				y(0),
-				width(0),
-				height(0),
-				used_in_frame(false),
-				msaa(RS::VIEWPORT_MSAA_DISABLED),
-				use_fxaa(false),
-				use_debanding(false),
-				used_dof_blur_near(false),
-				mip_maps_allocated(false),
-				clear_color(Color(1, 1, 1, 1)),
-				clear_requested(false) {
+
+				clear_color(Color(1, 1, 1, 1)) {
```

Comment:
This could also be initialized inline I think. If not, at least the empty line after `RenderTarget() :` should be removed.